conftest.py

```diff
@@ -724,21 +732,18 @@ def pytest_exception_interact(node: Item | Collector, call: CallInfo[Any], repor
             except Exception as db_exception:
                 test_start_time = 0
                 LOGGER.warning(f"Error: {db_exception} in accessing database.")
-            if not test_start_time:
-                since_time = TIMEOUT_5MIN
-                LOGGER.warning(
-                    f"Could not get start time of test: {test_name}. Collecting must-gather for {since_time}s"
-                )
-            else:
-                # if the test duration is 0 seconds, collect must-gather for past 60 seconds
-                since_time = (int(datetime.datetime.now().strftime("%s")) - test_start_time) or TIMEOUT_1MIN
+
             try:
                 collection_dir = os.path.join(get_data_collector_dir(), "pytest_exception_interact")
-                collect_default_cnv_must_gather_with_vm_gather(since_time=since_time, target_dir=collection_dir)
+                collect_default_cnv_must_gather_with_vm_gather(
+                    since_time=calculate_must_gather_timer(test_start_time=test_start_time), target_dir=collection_dir
+                )
                 if inspect_str:
                     target_dir = os.path.join(collection_dir, "inspect_collection")
                     inspect_command = (
-                        f"{INSPECT_BASE_COMMAND} {inspect_str} --since={since_time}s --dest-dir={target_dir}"
+                        f"{INSPECT_BASE_COMMAND} {inspect_str} "
+                        f"--since={calculate_must_gather_timer(test_start_time=test_start_time)}s "
```

Comment:
So you want to recalculate it here again, because some time has passed while we were collecting the must-gather in `collect_default_cnv_must_gather_with_vm_gather`?

Reply:
exactly, must-gather takes several minutes to get all necessary data, so we need to have "since_time" recalculated again